### DISCUSSION TITLE
Is there a big peformance boost with 4or 8 gigs of ram compared to 2gigs

### DISCUSSION THREAD

**Discussion by Neogx** [Geektoolkit/Dynaframe3#162] [Q&A]
I'm deciding which raspberry pi to buy and debating on which would work smoothly with the least slowdown. Just wondering what you all recommend for dynaframe. Will the 2 gig version work just as good as the 4 or 8 or will i see improvement by buying more ram

**Comment by Geektoolkit** [comment]
Honestly the thing that makes things chug is the CPU/GPU for the most part.  If you get a pi4 2 gig it should be fine, though if you want to run more stuff besides Dynaframe the 8 gig is a good way to go. That all being said, I'm working on a new version of Dynaframe (which will be called Dynaframe pro) and even with all of the optimizations I've made in the past 4-6 months, if you run it at 4k it still chugs. My recommendation for 4k is to turn off transitions and then use it as a web enabled video/image player. In the end the Pi is just only so powerful, so you hit it's limits quickly. 
  That all being said I run 1080p on 4k panels using the TV's upscaling, and I love the look of it and transitions run smooth (make sure you aren't VPN'd in when running it though!)
  
  If you're looking to run dedicated, I develop on a pi4 2gig for most of my work, but I only plan to run at 1080p

**Comment by qwksilver** [comment]
i run 4k images on a 4k panel on a 4b 2gb with 500ms transitions and it goes well, the transitions feel abrupt but not as bad as no transition at all, at 800+ you get a shutter effect as you fade, and at 1500+ it is really choppy.
